inline/src/inlines/render.rs

```diff
@@ -68,17 +67,17 @@ impl Render for TokenDelimiters {
             tags.push_str(close_tag);
         }
 
-        Ok(Html {
+        Html {
             body: tags,
             ..Default::default()
-        })
+        }
     }
 }
 
 impl Render for Inline {
-    fn render_html(&self) -> Result<Html, MappedLogId> {
+    fn render_html(&self) -> Html {
         let mut res = String::new();
-        let tags = self.delimiters().render_html()?.body;
+        let tags = self.delimiters().render_html().body;
         let mut tags = tags.split('|');
```

Comment:
Food for thought: `Html` right now can contain anything, we should maybe think about the design of that. 
One solution might be to introduce `HtmlElement` struct, and it should have open and close tags, attributes, and inner `Html`. We might even use existing crates for this one to be honest. This hack with splitting tags by some arbitrary delimiter is not optimal. (I think it was initially implemented by me, yuck 😆)

Reply:
😄 I do not remember what this code was supposed to do, but I think inlines is mostly your code 🤔 

Going for a crate that supports HTML syntax might be a good idea indeed.
Best case would be a crate that also validates input to be valid and safe HTML.